command/aarelayer/service/aa_pool.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"sync"
+)
+
+type AAPool interface {
```

Comment:
Can we add some comments in this file?

Reply:
This is just base, correct version of pool will be implemented via separated task

Reply:
Added https://github.com/0xPolygon/polygon-edge/pull/1252/commits/f8527534f60da34f423360164a5bfd3a2cbf29d9